ethstore/src/json/key_file.rs

```diff
@@ -46,7 +46,7 @@ pub struct KeyFile {
 	pub id: Uuid,
 	pub version: Version,
 	pub crypto: Crypto,
-	pub address: H160,
+	pub address: Option<H160>,
```

Comment:
~I'm not sure about this change either. I don't know how this affects runtime. Without `address` field we don't know that what account we just loaded and I am afraid that it may be simply unusable.~

nvm, I see that you load the password in `from_file`. Please describe in the code why this field is optional.